app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ReviewsController < ApplicationController
+  skip_before_action :authenticate_user!, only: [:index]
```

Comment:
You don't have an index method in this controller, so you don't need to skip this before action